catalog/app/containers/Admin/Users/Users.js

```diff
@@ -89,6 +89,11 @@ function Invite({ close, roles, defaultRoleId }) {
             username: 'invalid',
           }
         }
+        if (APIConnector.HTTPError.is(e, 400, /Username is invalid/)) {
```

Comment:
why not just adjust the regex above to merge these cases? and since we're not going to use it with the older registry, i think it's safe to keep only the current option

Reply:
> why not just adjust the regex above to merge these cases?

Because, it's easier to scan two strings instead of one regex

>  i think it's safe to keep only the current option

Ok, I removed old one